src/test/java/no/ssb/jsontemplate/templatetests/ObjectTest.java

```diff
@@ -13,26 +12,26 @@ class ObjectTest {
 
     @Test
     void test_twoLevelObject() {
-        DocumentContext document = parse(new JsonTemplate("{anObject:{aField:@s}}"));
+        DocumentContext document = TestUtils.parse(new JsonTemplate("{anObject:{aField:@s}}"));
```

Comment:
minor: Consider to static import TestUtils (as it was before). This also applies to other files in test

Reply:
The code cleaner in IntelliJ has removed the static import from all test files. I will keep it as it is for the moment.